src/it/resources/com/google/checkstyle/test/chapter7javadoc/rule713atclauses/InputNonEmptyAtclauseDescriptionCheck.java

```diff
@@ -34,7 +34,7 @@ public int foo2(String a, int b, double c) throws Exception
 	 * @param a
 	 * @param b
 	 * @param c
-	 * @return
```

Comment:
`@return` should be removed from `NonEmptyAtclauseDescriptionCheck`. Also [`@exception`](http://docs.oracle.com/javase/7/docs/technotes/tools/windows/javadoc.html#exception) which is a synonym for `@throws` should be added to default tokens for the check since `@throws` is a default token for it. 

Also, there's a potential for other tags to be added to this check but that depends on the philosophy of the check. 

@romani @Vladlis @rnveach Please look into this. Should an issue be created for this

Reply:
yes, please open an issue

Reply:
new issue should be created. 
It might be done on purpose previously as it was cleat that nobody in the works knows javadoc tag syntax.
as we become more strict in this area ... Checks should be updated, should be done completely separately after this PR is merged.